src/main/java/com/nutomic/syncthingandroid/fragments/DeviceSettingsFragment.java

```diff
@@ -97,13 +103,43 @@ public void onCreate(Bundle savedInstanceState) {
                 mDevice.Name = "";
                 mDevice.DeviceID = "";
                 mDevice.Addresses = "dynamic";
-                mDevice.Compression = true;
+                mDevice.Compression = "always";
                 ((EditTextPreference) mDeviceId).setText(mDevice.DeviceID);
             }
         }
     }
 
     /**
+     * Check for any upgrades of the preferences.
+     */
+    private void checkUpgradePreferences(PreferenceFragment activity) {
```

Comment:
Hmm, that explains why it took me that long to read the preference value:)
Removed.